ansible/kill.yml

```diff
@@ -30,7 +30,7 @@
     - name: "stop mesos slaves"
       service: name=mesos-slave state=stopped
       when: "'mesosmaster' in groups"
-- hosts: all
+- hosts: all:!{{ azure_proxy_host }}
```

Comment:
Why is this needed?

Reply:
The tasks within docker.yml install and bootstrap docker / swarm only on the workers and on swarmmanager hosts. But when kill / wipe are run, the above task to stop Docker would fail on `all` because in Azure there's a separate VM which gets setup as a proxy host (and that is not assigned a `worker` or `swarmmanager` role. So this is just addressing a dormant issue which came up when I tested the fix for the main issue in this thread.